libs/wingsdk/src/simulator/client.ts

```diff
@@ -20,17 +20,12 @@ export function makeSimulatorClient(url: string, handle: string) {
         // import undici dynamically to reduce the time it takes to load Wing SDK
         // undici is used instead of the built-in fetch so that we can customize the
         // keep-alive and headers timeouts to be 15 minutes instead of the default 5 seconds
-        const { fetch, Agent } = await import("undici");
+        const { fetch } = await import("undici");
         resp = await fetch(url + "/v1/call", {
           method: "POST",
           headers: { "Content-Type": "application/json" },
           body: JSON.stringify(body),
-          dispatcher: new Agent({
-            keepAliveTimeout: 15 * 60 * 1000,
-            keepAliveMaxTimeout: 15 * 60 * 1000,
-            headersTimeout: 15 * 60 * 1000,
-            bodyTimeout: 15 * 60 * 1000,
-          }),
+          keepalive: false,
```

Comment:
IIRC one of the reasons I brought in undici was to customize some of these other fields that weren't available with the built-in fetch -- if we don't need them it could be worth trying to use the built-in `http` module instead

Reply:
fetch is still considered experimental in node versions we support and it will warn the users if they're not on a new enough version, so I think keeping undici for now is ok